tests/streams/async-stream.js

```diff
@@ -33,4 +33,16 @@ describe('Async Stream', () => {
         asyncStream.end();
     });
 
+    it('should re-emit errors when fragment stream emit errors', (done) => {
+        const asyncStream = new AsyncStream();
+        const st1 = new stream.PassThrough();
+        asyncStream.on('error', (err) => {
+            assert.equal(err.message, 'blah');
```

Comment:
"blah" is a bit unprofessional in this context :) Maybe "TestError" is a better choice.